src/ert/analysis/_es_update.py

```diff
@@ -177,6 +182,9 @@ def _compute_observation_statuses(
      * status of (the responses of) each observation,
        corresponding to ObservationStatus
     """
+    std_cutoff = outlier_settings.std_cutoff
+    alpha = outlier_settings.alpha
```

Comment:
Only used once each? Could also just use the property directly instead of creating variable.